src/tools/rest-frontend/public/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html ng-app="elektra.rest.angular">
+
+    <head>
+
+		<!-- Application base path (i.e. subfolder of URL where application is reachable, mostly /) -->
```

Comment:
I will try to fix the white-space with the IDE first. Haven't really found a good solution for JS yet, HTML is not the problem normally.